src/renderer/components/+custom-resources/crd-details.tsx

```diff
@@ -133,6 +133,6 @@ export class CRDDetails extends React.Component<Props> {
   }
 }
 
-apiManager.registerViews(crdApi, {
+apiManager.registerViews(crdBetaApi, {
```

Comment:
are there any difference between using `crdApi` or `crdBetaApi` here?

Reply:
Sure, I don't quite know how the api is used, if it used to list the associated contents, then changing it might do something.